language/move-model/src/ty.rs

```diff
@@ -357,36 +334,26 @@ impl Type {
             Vector(et) => et.is_incomplete(),
             Reference(_, bt) => bt.is_incomplete(),
             TypeDomain(bt) => bt.is_incomplete(),
-            Error | Primitive(..) | TypeLocal(..) | TypeParameter(_) | ResourceDomain(..) => false,
+            Error | Primitive(..) | TypeParameter(_) | ResourceDomain(..) => false,
         }
     }
 
     /// Return true if this type contains generic types (i.e., types that can be instantiated).
     /// A caller can have finer-grained control on what can be considered as a generic type.
-    pub fn is_generic(
-        &self,
-        treat_type_param_as_open: bool,
-        treat_type_local_as_open: bool,
-    ) -> bool {
+    pub fn is_generic(&self, treat_type_param_as_open: bool) -> bool {
```

Comment:
I think we can remove this `treat_type_param_as_open` as well. They should always be considered `true`.

Or rather, this function can be removed too, and merge with `is_open`.

Reply:
Done